tests/conftest.py

```diff
@@ -54,3 +54,25 @@ def example_ingest_wrong_format() -> FileStorage:
     """An example pre ingest spreadsheet in towns-fund format."""
     with open(UnitTestConfig.EXAMPLE_INGEST_WRONG_FORMAT, "rb") as file:
         yield file
+
+
+@pytest.fixture()
+def mocked_custom_email_auth(monkeypatch):
+    def custom_access_token(return_app=SupportedApp.POST_AWARD_SUBMIT, auto_redirect=True):
+        return {"accountId": "test-user", "roles": [], "email": "madeup@madeup.gov.uk"}
+
+    monkeypatch.setattr(
+        "fsd_utils.authentication.decorators._check_access_token",
+        custom_access_token,
+    )
+
+
+@pytest.fixture()
+def flask_test_client_custom_email(mocked_custom_email_auth) -> FlaskClient:
+    """
+    Creates the test client with custom email authentication.
+
+    :return: A flask test client.
+    """
+    with create_app(config.Config).test_client() as test_client:
+        yield test_client
```

Comment:
I don't think this will be reused anywhere and I think it's clearer to just overwrite the original auth'd mocked access token with an unauthorized one in the unit test.

We can use these lines in the `test_unauthorised_user` test instead. 
```
def custom_access_token(return_app=SupportedApp.POST_AWARD_SUBMIT, auto_redirect=True):
        return {"accountId": "test-user", "roles": [], "email": "madeup@madeup.gov.uk"}

    monkeypatch.setattr(
        "fsd_utils.authentication.decorators._check_access_token",
        custom_access_token,
    )
```

Reply:
Removed flask_test_client_custom_email, but couldn't overwrite the email in the test function itself so left mocked_custom_email_auth in conftest and added it as a parameter in the test.